src/main/java/uk/gov/mca/beacons/api/services/DeleteBeaconService.java

```diff
@@ -25,25 +28,36 @@
   private final BeaconGateway beaconGateway;
   private final UserGateway userGateway;
   private final NoteGateway noteGateway;
+  private final OwnerGateway ownerGateway;
+  private final EmergencyContactGateway emergencyContactGateway;
+  private final UseGateway useGateway;
 
   @Autowired
   public DeleteBeaconService(
     BeaconGateway beaconGateway,
     UserGateway userGateway,
-    NoteGateway noteGateway
+    NoteGateway noteGateway,
+    OwnerGateway ownerGateway,
+    EmergencyContactGateway emergencyContactGateway,
+    UseGateway useGateway
   ) {
     this.beaconGateway = beaconGateway;
     this.userGateway = userGateway;
     this.noteGateway = noteGateway;
+    this.ownerGateway = ownerGateway;
+    this.emergencyContactGateway = emergencyContactGateway;
+    this.useGateway = useGateway;
   }
 
   public void delete(DeleteBeaconRequestDTO request) {
     final User user = userGateway.getUserById(request.getUserId());
     if (user == null) throw new UserNotFoundException();
 
+    final var beaconIdToBeDeleted = request.getBeaconId();
+
     final Note note = Note
       .builder()
-      .beaconId(request.getBeaconId())
+      .beaconId(beaconIdToBeDeleted)
       .email(user.getEmail())
       .fullName(user.getFullName())
```

Comment:
I would remove these two lines as we shouldn't be capturing PII for the note when we delete a beacon.

@stuart-madetech FYI

Reply:
You should be able to update the unit tests to check that the full name/email are null 😄

Reply:
Do you mean line 61 & 62 (email & full name)?

Reply:
changes made